tutorials/networking/high_level_multiplayer.rst

```diff
@@ -239,6 +296,23 @@ The function ``multiplayer.get_remote_sender_id()`` can be used to get the uniqu
         var sender_id = multiplayer.get_remote_sender_id()
         # Process the input and affect game logic.
 
+ .. code-tab:: csharp
+
+    public void OnSomeInput() // Connected to some input
```

Comment:
```suggestion
    public void OnSomeInput() // Connected to some input.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).